exporter/clickhouseexporter/example/default_ddl/histogram_metrics.sql

```diff
@@ -0,0 +1,42 @@
+-- Default Histogram metrics table DDL
+
+CREATE TABLE IF NOT EXISTS otel_metrics_histogram (
```

Comment:
I think we need a unit test to verify the table DDL gennerate by code is as same as the example ones. 

it can prevent the example ones allway up-to-date.

Reply:
I considered reading the template strings from a file to avoid this completely, but ultimately this DDL is just an example.
As long as the column names and types are the same, the inserts should run correctly.

If we were to add unit tests then I would prefer to just read the SQL in from a file, or maybe embed it in the code at build time, or extract them into individual `.go` files. Let me know if any of this sounds like a better solution and I will update the PR. 👍